### ISSUE TITLE
Number of batches

### ISSUE THREAD

**Issue by UditGupta10** [thunlp/OpenKE#100]
In train.py we need to set number of batches; shouldn't we set the batch size instead (and number of batches calculated accordingly). We have no control over batch_size which is an important thing for gradient descent. For a large dataset, if we set number of batches to default 100 the batch size will be very large.
Please comment why one should be preferred over another.

**Comment by ShulinCao** [comment]
We just follow the original papers and set the number of batches. Of course you can set the batch size, and we believe there is no difference because nbatches * batch_size = dataset_size. If you know the size of your dataset, you can choose the proper nbatches for a proper batch_size.
